tools/BUILD.bazel

```diff
@@ -22,14 +22,10 @@ exports_files(["CROSSTOOL"])
     name = "cc-gcc-" + cpu,
     all_files = ":empty",
     compiler_files = ":empty",
-    cpu = cpu,
     dwp_files = ":empty",
-    dynamic_runtime_libs = [":empty"],
```

Comment:
the `_runtime_libs` attributes changed names, but they're also optional now, so I just dropped them.